### PR TITLE
support stddev_pop function

### PR DESCRIPTION
**Pull request by liujiayi771**
## What changes were proposed in this pull request?

support stddev_pop function, velox PR https://github.com/oap-project/velox/pull/129

## How was this patch tested?

run stddev_pop unit test in DataFrameAggregateSuite.

### CODE REVIEW

**Review by rui-mo — COMMENTED** [review]
@liujiayi771 Thanks for your contribution. Can we add some Scala unit test to check the result, and also make sure the plan is successfully converted to `HashAggregateExecBaseTransformer`?

**Comment by liujiayi771** [comment]
> 

I added a test similar to stddev_samp in the TestOperator. Some correctness tests have been included in GlutenDataFrameAggregateSuite, such as "stddev", "zero stddev", can we test the correctness based on these unit tests?

**Comment by rui-mo** [comment]
> I added a test similar to stddev_samp in the TestOperator. Some correctness tests have been included in GlutenDataFrameAggregateSuite, such as "stddev", "zero stddev", can we test the correctness based on these unit tests?

Yes, thank you. The test added in this PR also compares the result with vanilla Spark.

**Comment by liujiayi771** [comment]
> 

@rui-mo The unit test failed because the code modified by velox has not been merged.

**Comment by rui-mo** [comment]
> @rui-mo The unit test failed because the code modified in velox has not been merged.

@liujiayi771 I see, and we need to make sure the Github actions can pass before merge. Can we change the used Velox branch to yours for testing?  They are at: [get_velox](https://github.com/oap-project/gluten/blob/main/ep/build-velox/src/get_velox.sh#L5-L6) and [build_velox](https://github.com/oap-project/gluten/blob/main/ep/build-velox/src/build_velox.sh#L13-L14).

File: ep/build-velox/src/build_velox.sh [hunk 1]
```diff
@@ -10,8 +10,8 @@ BUILD_PROTOBUF=ON
 BUILD_TYPE=release
 VELOX_HOME=
 #for ep cache
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/liujiayi771/velox.git
+VELOX_BRANCH=stddev_pop
```
Comment (rui-mo) [inline ep/build-velox/src/build_velox.sh:14]: Velox PR has been merged, and this change could be reverted, thanks.

File: ep/build-velox/src/get_velox.sh [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/liujiayi771/velox.git
+VELOX_BRANCH=stddev_pop
```
Comment (rui-mo) [inline ep/build-velox/src/get_velox.sh:6]: Same.

File: ep/build-velox/src/build_velox.sh (hunk 1 above)
Comment (liujiayi771) [reply ep/build-velox/src/build_velox.sh:14]: ok.

File: ep/build-velox/src/get_velox.sh (hunk 2 above)
Comment (liujiayi771) [reply ep/build-velox/src/get_velox.sh]: I have reverted these changes. By the way, I will support the two functions var_samp and var_pop later.

**Comment by liujiayi771** [comment]
@rui-mo Thanks for helping me review the code, I have reverted these changes. By the way, I will support the two functions var_samp and var_pop in another PR.

**Review by rui-mo — APPROVED** [review]

**Pull request merged**